homeassistant/components/google_photos/manifest.json

```diff
@@ -6,6 +6,6 @@
   "dependencies": ["application_credentials"],
   "documentation": "https://www.home-assistant.io/integrations/google_photos",
   "iot_class": "cloud_polling",
-  "loggers": ["googleapiclient"],
-  "requirements": ["google-api-python-client==2.71.0"]
+  "loggers": ["google_photos_library_api"],
```

Comment:
It still uses `google-api-python-client` underwater right? Should we still add the logger?

Reply:
No it does not --  I didn't just copy out the code, i re-wrote it using aiohttp. (though i see its accidentally still listed as a library dependency).